README.md

```diff
@@ -10,7 +10,7 @@
 This package is a PHP port of the `java.util.Optional` class in the 
 [OpenJDK](http://hg.openjdk.java.net/lambda/lambda/jdk/file/tip/src/share/classes/java/util/Optional.java) libraries.
 
-Currently maintained by [Niklas Schöllhorn](https://github.com/mrpixeldream), taken over 3rd March, 2019
+Currently maintained by [Niklas Schöllhorn](https://github.com/nschoellhorn), taken over 3rd March, 2019
```

Comment:
Oh, what's up with this?

Reply:
I'm gonna change my Github username today to something a little bit more mature. Went with this kiddy name long enough. :)

Reply:
Meh, I have my kiddy name since 22 years :-P

Reply:
But at least it reflects your real name somehow :) and it doesn't contain "mr" and "dream"

Reply:
Still works, IMO. Your choice though :-P

Reply:
Yeah, gonna change as soon as we got the changes merged.  
Do you have any clue why Scrutinizer doesn't want to run? Says something like "misuse of shell builtins".

Reply:
Ha, seems like it had something to do with the phpcs config in Scrutinizer. According to the docs, using it via "tools" is legacy. Since we use it in Travis anyways, I removed it from the Scrutinizer build.